### PR TITLE
Fixes: #864 Aligns the forgot password to right in Android

### PR DESCRIPTION
**Pull request by saketkumar95**
fixes : #864

Kind of reverting this PR #811 

iOS. 
Current master : 

screen shot 2017-07-11 at 3 22 18 am

Android : 

screenshot_20170711-032243

This PR : 

iOS:

screen shot 2017-07-13 at 5 50 56 am

Android : 
19911782_1420150728061837_1835120685_o

### CODE REVIEW

**Comment by borisyankov** [comment]
It looks fine as it is right now.

**Comment by saketkumar95** [comment]
But in android it is not align to the right most it does only in the iOS.

**Comment by saketkumar95** [comment]
@borisyankov Updated! Ready for review. 👍

File: src/styles/theme.js [hunk 1]
```diff
@@ -77,9 +77,10 @@ export default ({ color, backgroundColor, borderColor }: Props) => ({
     alignItems: 'stretch',
   },
   link: {
-    marginTop: 10,
+    paddingTop: 10,
     fontSize: 15,
     color: BRAND_COLOR,
+    textAlign: 'right',
```
Comment (vishwesh3) [inline src/styles/theme.js]: Now whole row becomes clickable, then what is the purpose of aligning text to the right. Can we limit touchable area to the text itself.
ezgif com-video-to-gif-2

Comment (saketkumar95) [inline src/styles/theme.js]: Done! 👍

Comment (saketkumar95) [inline src/styles/theme.js]: Actually It was clickable on the master too. If you can check, on iOS and Android both were clickable.

**Comment by saketkumar95** [comment]
@borisyankov Updated.
@vishwesh3 Do review, so that now there will not be any such errors left.

**Pull request merged**